### ISSUE TITLE
versioned alpha release branches compatible with 0.22, 0.23, 0.24 versions of mesos

### ISSUE THREAD

**Issue by spacejam** [mesosphere/etcd-mesos#42]
due to mesos-go handling a single version of mesos, we need to have different releases for each mesos version.  For our initial alpha open-source release, we need to create a branch that is based on each version of mesos-go.

blockers:
1. mesos-go compatible with 0.23
2. mesos-go compatible with 0.24

**Comment by pires** [comment]
I strongly suggest to drop this in favor of supporting Mesos 1.x and the v1 API in the short term.

/cc @jdef

**Comment by jdef** [comment]
yes for v1 support. there's still the issue of tracking mesos compatibility though. maybe just re-title the issue?
